backend/LexData/Entities/UserEntityConfiguration.cs

```diff
@@ -25,17 +26,29 @@ public override void Configure(EntityTypeBuilder<User> builder)
 
 public static class UserEntityExtensions
 {
-    public static Expression<Func<User, bool>> FilterByEmail(string email)
+    public static Expression<Func<User, bool>> FilterByEmailOrUsername(string emailOrUsername)
     {
-        return user => user.Email == email || user.Username == email;
+        return user => user.Email == emailOrUsername || user.Username == emailOrUsername;
     }
-    public static IQueryable<User> FilterByEmail(this IQueryable<User> users, string email)
+    public static IQueryable<User> FilterByEmailOrUsername(this IQueryable<User> users, string? emailOrUsername)
     {
-        return users.Where(FilterByEmail(email));
+        if (emailOrUsername is null) return Enumerable.Empty<User>().AsQueryable();
```

Comment:
I don't think there's anywhere we're calling this function with a null email, so we could just simplify this code to not allow calling with null.

Reply:
Done in commit 352e13da.